prac_05/state_names.py

```diff
@@ -8,10 +8,16 @@
                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
 
-state_code = input("Enter short state: ")
+state_code = input("Enter short state: ").upper()
 while state_code != "":
-    if state_code in CODE_TO_NAME:
+    try:
         print(state_code, "is", CODE_TO_NAME[state_code])
-    else:
+    except KeyError:
         print("Invalid short state")
     state_code = input("Enter short state: ")
+
+
+max_code_length = max(len(code) for code in CODE_TO_NAME.keys())
+max_name_length = max(len(name) for name in CODE_TO_NAME.values())
```

Comment:
Looks pretty good, but I'm not sure if you need to find the max_name_length for the values as it doesn't affect the formatting, I think would just start printing from the "is" anyway :).